pkg/scheduler/framework/v1alpha1/interface.go

```diff
@@ -262,8 +262,7 @@ type FilterPlugin interface {
 	// Filter is called by the scheduling framework.
 	// All FilterPlugins should return "Success" to declare that
 	// the given node fits the pod. If Filter doesn't return "Success",
-	// please refer scheduler/algorithm/predicates/error.go
-	// to set error message.
+	// it will return "Unschedulable" or "Error".
```

Comment:
Actually, you missed "UnschedulableAndUnresolvable". Could you please add it?
Also, use present tense.

Reply:
ok :)